dashy/src/app/dashboard/widget/divider-widget/divider-widget.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="col-md-12">
+  <h2>{{title}}</h2>
+  <h5>&nbsp;&nbsp;&nbsp;{{description}}</h5>
```

Comment:
How about adding a `border-bottom` in combination with some `padding` to the `h2` tag instead of using a `h2` and a `hr`.  This will save some valueable space.